domains/zishu.is-cool.dev.json

```diff
@@ -0,0 +1,22 @@
+{
+    "description": "Will be used on my github.io website that I working on.",
+    
+    "domain": "is-cool.dev",
+    "subdomain": "zishu",
+    
+    "owner": {
+        "repo": "https://github.com/HP-L/hp-l.github.io",
+        "email": "lake.hupo.2001@outlook.com"
+    },
+    
+    "record": {
+        "NS": [
```

Comment:
We do not do any NS records, check the last box on your PR:

![image](https://github.com/user-attachments/assets/8acfa19c-8c94-49d5-b941-50ce20f81c74)